packages/fx-core/src/plugins/resource/bot/plugin.ts

```diff
@@ -155,8 +156,12 @@ export class TeamsBotImpl {
       group_name
     );
 
-    await handler?.next(ProgressBarConstants.SCAFFOLD_STEP_UNZIP);
-    zipContent.extractAllTo(this.config.scaffold.workingDir!, true);
+    try {
+      await handler?.next(ProgressBarConstants.SCAFFOLD_STEP_UNZIP);
```

Comment:
move `handler?.next` outside of the try block. The block should not cover its error.